pdb2sql/StructureSimilarity.py

```diff
@@ -135,8 +137,8 @@ def compute_lrmsd_fast(self, lzone=None, method='svd', check=True):
                 self.ref, resData, return_not_in_zone=True)
 
         # get the translation so that both A chains are centered
```

Comment:
This part could be replaced by a generic `superpose` function.

Reply:
done. I'm not sure the name of the generic function is ideal ('transfor_xyz_from_selection_superposition') .. any input welcome here